pymystrom/switch.py

```diff
@@ -48,15 +48,21 @@ async def get_state(self) -> None:
         """Get the details from the switch/plug."""
         url = URL(self.uri).join(URL("report"))
         response = await request(self, uri=url)
-        self._consumption = response["power"]
-        self._consumedWs = response["Ws"]
+        try:
+            self._consumption = response["power"]
+        except KeyError:
+            self._consumption = 0
+        try:
+            self._consumedWs = response["Ws"]
+        except KeyError:
+            self._consumedWs = 0
         self._state = response["relay"]
         try:
             self._temperature = response["temperature"]
         except KeyError:
             self._temperature = None
 
-        url = URL(self.uri).join(URL("info.json"))
+        url = URL(self.uri).join(URL("api/v1/info"))
```

Comment:
This will break switches with hardware revision v1. Or at least the firmware release 2.68.10 is still using `info.json`.

Reply:
See pull request #34, there is a fallback to the old info.json URL implemented, and tested with Switch Zero and the Switch.